config-service-impl/src/main/java/org/hypertrace/config/service/ConfigServiceUtils.java

```diff
@@ -59,4 +62,22 @@ public static Value merge(Value defaultConfig, Value overridingConfig) {
   public static String getActualContext(String rawContext) {
     return Strings.isNullOrEmpty(rawContext) ? DEFAULT_CONTEXT : rawContext;
   }
+
+  private static boolean isNull(Value value) {
+    if (value == null) {
+      log.error("NULL Value encountered. This is unexpected and indicates a BUG in code.",
+          new RuntimeException());
```

Comment:
does the error logger require an exception to the message? Doesn't look like it adds any value

Reply:
I just thought in case we hit this, the stack trace can help pin point the place where we're seeing a null value, which can probably help us debug quickly.